redis/_cache.py

```diff
@@ -224,12 +222,17 @@ def __init__(
         self.key_commands_map = defaultdict(set)
         self.commands_ttl_list = []
 
-    def set(self, command: str, response: ResponseT, keys_in_command: List[KeyT]):
+    def set(
+        self,
+        command: Union[str, Iterable[str]],
```

Comment:
Why it need to be also Iterable[str]?

Reply:
Because in all unit tests tuples are being sent for this parameter.

Reply:
Not in unit tests, that is for the `get`. In Connection we have this signature for example:

    def _add_to_local_cache(
        self, command: Tuple[str], response: ResponseT, keys: List[KeysT]
    ):

Reply:
So maybe we need only Iterable[str]? I'm not sure...

Reply:
This is a broader discussion, I think we can leave it with both for the beta release. In general I think we should not expose to the outside world the way we represent commands in the cache. We should only expose the concept of keys externally, and do the mapping to commands and back only internally, and then we have full freedom of representation.

Reply:
I used `Sequence` instead of `Iterable` and `Tuple`. I think it is the most expressive one.